src/main/java/organice/model/person/Name.java

```diff
@@ -28,7 +28,12 @@
     public Name(String name) {
         requireNonNull(name);
         checkArgument(isValidName(name), MESSAGE_CONSTRAINTS);
-        fullName = name;
+        String[] nameArr = name.trim().split(" ");
+        String fullName = "";
+        for (String str : nameArr) { //Make the first character in name uppercase the rest lowercase
+            fullName += str.substring(0, 1) + str.substring(1).toLowerCase() + " ";
+        }
+        this.fullName = fullName.trim();
```

Comment:
Mr Francesco de Freta will be wrongfully uppercased to Francesco De Freta while Dr Ronald McDonald will have his name changed to Ronald Mcdonald. I think we should leave this part to the users to input properly.